aws/rust-runtime/aws-config/src/environment/shared_timeout_config.rs

```diff
@@ -5,61 +5,60 @@
 
 //! Load timeout configuration properties from environment variables
 
-use aws_smithy_types::timeout::{parse_str_as_timeout, TimeoutConfig, TimeoutConfigError};
+use aws_smithy_types::timeout::{
+    parse_str_as_timeout, SharedTimeoutConfig, SharedTimeoutConfigError,
+};
 use aws_types::os_shim_internal::Env;
 use std::time::Duration;
 
-const ENV_VAR_CONNECT_TIMEOUT: &str = "AWS_CONNECT_TIMEOUT";
-const ENV_VAR_TLS_NEGOTIATION_TIMEOUT: &str = "AWS_TLS_NEGOTIATION_TIMEOUT";
-const ENV_VAR_READ_TIMEOUT: &str = "AWS_READ_TIMEOUT";
+// const ENV_VAR_CONNECT_TIMEOUT: &str = "AWS_CONNECT_TIMEOUT";
+// const ENV_VAR_TLS_NEGOTIATION_TIMEOUT: &str = "AWS_TLS_NEGOTIATION_TIMEOUT";
+// const ENV_VAR_READ_TIMEOUT: &str = "AWS_READ_TIMEOUT";
```

Comment:
should probably note why these are commented out, maybe with link to a github issue